omnibus/config/software/datadog-agent-dependencies.rb

```diff
@@ -31,7 +31,7 @@
 # Include traps db file in snmp.d/traps_db/
 dependency 'snmp-traps'
 
-dependency 'secret-generic-connector'
+dependency 'secret-generic-connector' if !heroku_target?
```

Comment:
```suggestion
dependency 'secret-generic-connector' unless heroku_target?
```

Reply:
otherwise I think rubocop or similar will complain sooner or later. That being said, that's definitely a nit and non blocking

Reply:
Too late I already accepted ;)